wbooks/src/app/components/BookDetail/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './styles'
+
+export default function BookDetail({title, subtitle, url}: {title: string, subtitle: string, url: string}) {
```

Comment:
1) Extract TS interface:
```
interface Book {
  title: string;
  subtitle: string;
  url: string
}

function BookDetail({ title, subtitle, url }: Book) {...}
```
2) First define `function BookDetail({ title, subtitle, url }: Book) {...}` and then, at the bottom of the file, `export default BookDetail;`